sklearn/neighbors/graph.py

```diff
@@ -44,6 +65,11 @@ def kneighbors_graph(X, n_neighbors, mode='connectivity', metric='minkowski',
         The default distance is 'euclidean' ('minkowski' metric with the p
         param equal to 2.)
 
+    include_self: bool, default None
```

Comment:
Hmmm... I'm almost inclined to not do that, because the distance mode was include_self=broken, not False. But you're right. Still "None" tells the user nothing. Tell them instead, default is backwards-compatible, or default=True iff mode='connectivity'.